eng/build.ps1

```diff
@@ -0,0 +1,48 @@
+[CmdletBinding(PositionalBinding=$false)]
+Param(
+  [string] $ArchGroup,
+  [switch] $Release,
+  [Alias("framework")][string] $TargetGroup,
+  [Alias("allconfigurations")][switch] $BuildAllConfigurations,
+  [switch] $OuterLoop,
+  [switch] $SkipTests,
+  [Alias("sync")][switch] $Restore,
+  [switch] $Clean,
+  [Alias("tests")][switch] $BuildTests,
+  [switch] $Coverage,
+  [Parameter(ValueFromRemainingArguments=$true)][String[]]$ExtraArgs
+)
```

Comment:
I don't see an option here for which OS(Group).  Is it all OSGroups now by default, or is it something obscure to build managed Linux on Windows?

Reply:
I didn't add the shortcut because I didn't find any usages. We can always add the alias if folks want but they can always pass `/p:OSGroup=Linux` if they want.

Reply:
I'm good either way, just need(ed) to know how to adjust my build flow.